core/javaAction/init.py

```diff
@@ -18,12 +15,12 @@
     encoded = sys.argv[1]
 
 payload = {
```

Comment:
When running the actionProxy on localhost, do we want to allow `./invoke.py run` with no dictionary parameter after `run`?